src/olympia/signing/serializers.py

```diff
@@ -11,7 +11,7 @@
 from olympia.files.models import FileUpload
 
 
-class FileUploadSerializer(serializers.ModelSerializer):
+class SigningFileUploadSerializer(serializers.ModelSerializer):
```

Comment:
I'll be adding a new serializer for `FileUpload` instances (in files/serializers) so having this named the same thing but in a different place was potentially confusing